tests/functional/common.bash.in

```diff
@@ -386,6 +386,11 @@ function setup_common(){
 
 # Cleanup anything generated by this script
 function cleanup_common(){
+	# kill shim if it is still running
```

Comment:
@devimc Agreed, we should at least make that a warning. If cc-shim is still running at that point, this is a bug we should know about.